org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/wizards/AbapGitWizardPageRepositoryAndCredentials.java

```diff
@@ -248,12 +267,56 @@ public boolean validateAll() {
 				if (!fetchExternalRepoInfo()) {
 					return false;
 				}
+				else {
+					askUserToStoreCredentials();
+				}
```

Comment:
The `getNextPage( )` method gets executed before the `validateAll( )`. We would want to ask the user if the credentials are to be stored after the user clicks Next on credentials page and after `validateAll( )` is called. Since, `validateAll( )` is the last action that happens on the click of Next Button, it seems to be the best place for this logic to me.
For now I have moved the logic to `fetchExternalRepoInfo( )`. Please review it and suggest a better place in the code for this logic otherwise.